src/TiledArray/tiledarray.cpp

```diff
@@ -127,3 +128,16 @@ void TiledArray::ta_abort(const std::string &m) {
   std::cerr << m << std::endl;
   ta_abort();
 }
+
+
+void TiledArray::set_taskq_wait_busy() {
```

Comment:
these really should be free functions in madness namespace, I think ... fine otherwise